[msx] test Started org.apache.hadoop.hdfs.server.datanode.TestDirectoryScanner#testExceptionHandlingWhileDirectoryScan
2019-12-31 20:35:20,989 [Thread-184] INFO  namenode.NameNode (NameNode.java:performReconfig(961)) - [msx-restart] NameNode init, reconfigMode is none, do nothing
2019-12-31 20:35:21,157 [Thread-184] INFO  namenode.NameNode (NameNode.java:<init>(1039)) - [msx-restart] NameNode start
2019-12-31 20:35:21,169 [Thread-184] INFO  datanode.DataNode (DataNode.java:performReconfig(463)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2019-12-31 20:35:21,229 [Thread-184] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2732)) - [msx-restart] DataNode start
2019-12-31 20:35:22,264 [Thread-184] INFO  datanode.DataNode (DataNode.java:shutdown(2072)) - [msx-restart] DataNode stop, double check before stop, dfs.block.access.token.enable = false
2019-12-31 20:35:22,425 [Thread-184] INFO  namenode.NameNode (NameNode.java:stop(1082)) - [msx-restart] NameNode stop, double check before stop, dfs.block.access.token.enable = false
[msx] test Finished org.apache.hadoop.hdfs.server.datanode.TestDirectoryScanner#testExceptionHandlingWhileDirectoryScan test result = 1
